lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -640,6 +640,10 @@ def write(chunk)
             # field.
             preserved_keys = [
```

Comment:
Can you add a reminder to keep this list up to date with the user-facing documentation at https://cloud.google.com/logging/docs/agent/configuration#special-fields ?

Reply:
Sure thing. Done.